### PR TITLE
Containerize enriched content

### PR DESCRIPTION
**Pull request by BerniVarga**

### CODE REVIEW

File: document-store-api-sidekick@.service [hunk 1]
```diff
@@ -8,9 +8,8 @@ ExecStart=/bin/sh -c "\
   etcdctl set   /ft/services/document-store-api/healthcheck     true; \
   etcdctl mkdir /ft/services/document-store-api/servers; \
   etcdctl set /ft/healthcheck/document-store-api-%i/path /__health; \
-  etcdctl set /ft/healthcheck/document-store-api-%i/categories read; \
```
Comment (catalin-ilea) [inline document-store-api-sidekick@.service]: Any reason why you removed the read category?

Comment (BerniVarga) [inline document-store-api-sidekick@.service]: No. It's the results of a poor clean-up after a series of tests..:"> I put it back.

**Comment by catalin-ilea** [comment]
lgtm

**Comment by Zapadlo** [comment]
+1

**Comment by catalin-ilea** [comment]
lgtm

**Pull request merged**